src/corps-support.hbs

```diff
@@ -0,0 +1,36 @@
+---
+fullTitle: Gradebook Corps of Cadets Support
+description: We're here to help
```

Comment:
```suggestion
description: We're here to help
eleventyExcludeFromCollections: true
```

we need to add this to the thank you page as well - prevents the page from being indexed